spec/models/comment_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  @user = User.new(name: 'Tom', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Mexico.')
+  subject { Comment.new(text: 'hello man') }
+
+  before { subject.save }
+
+  it 'comments should be not be Valid' do
+    subject.text = nil
+    expect(subject).to_not be_valid
+  end
+  it 'Post comments counter can be set' do
+    subject.post = Post.new(author: @user, title: 'Post One', text: 'This is the post one')
+    subject.send(:comments_counter)
+    expect(subject.post.comments_counter).to eq(1)
+  end
+end
```

Comment:
- [ ] As requested by the previous reviewer 👇🏻 Kindly make sure to use `describe` as required in this milestone 👍🏻 

> - [ ] ... Please use `describe` as required in this milestone. See the example down below!
> ```ruby
> describe 'validations' do
>   it { should validate_presence_of(:name) }
> end
> ```

_Originally posted by @MahmoudBakr23 in https://github.com/MohamedCK/Blog_App/pull/3#discussion_r1089272303_

Reply:
Hi @BenMukebo, Thanks for the reminder I have added. I will be working on the `MIT` and the `Readme`.  👍🏽